tests/functional/olp-cpp-sdk-dataservice-read/DataserviceReadVolatileLayerClientTest.cpp

```diff
@@ -224,4 +341,244 @@ TEST_F(VolatileLayerClientTest, GetPartitionsDifferentFetchOptions) {
   }
 }
 
+TEST_F(VolatileLayerClientTest, Prefetch) {
+  const auto kTileId = GetTestTile();
+  auto layer = CustomParameters::getArgument(kPrefetchLayer);
+  auto catalog = CustomParameters::getArgument(kPrefetchCatalog);
+
+  PrepareForPrefetch();
```

Comment:
Can this be done in fixture SetUp?

Reply:
Prefetch tests need different setup. They use another catalog, credentials etc. + we need to write tiles data used only by these tests, so moving it to SetUp() will lead to overhead for other test.